cpp/src/arrow/compute/kernels/scalar_arithmetic_test.cc

```diff
@@ -43,8 +42,20 @@
 namespace arrow {
 namespace compute {
 
-template <typename T>
-class TestUnaryArithmetic : public TestBase {
+// InputType - OutputType pairs
```

Comment:
I don't see any pairs here?

Reply:
WTF was I thinking with this comment! Ughh.